packages/client/src/pages/ShoppingCartData/ShoppingCartData.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, Stack, Step, StepLabel, Stepper } from '@mui/material';
+
+import { OrderSummary } from '../../components';
```

Comment:
Jeszcze jedna rzecz, którą zapomniałem zaznaczyć, tutaj linter pokazuje niepotrzebny import, chyba niechcący to tu zostało.